src/tests/ftest/erasurecode/online_rebuild_single.yaml

```
@@ -33,6 +33,9 @@ pool:
   svcn: 1
   control_method: dmg
   pool_query_timeout: 30
+  rf: !mux
+    rf1:
+      properties: rd_fac:2
```

We don't need mux since it's only one variant
```suggestion
  properties: rd_fac:2
```

---

ah, thanks! @daltonbohning